### PR TITLE
Initial experiments

### PR DESCRIPTION
**Pull request by whitep4nth3r**

### CODE REVIEW

File: web/index.ts [hunk 1]
```diff
@@ -1,419 +1,430 @@
...
+            value: fontSize,
+            onchange: (val: string) => setLoadingState({ fontSize: val }),
+          }),
+        }),
+        H(Field, {
+          label: "Text Type",
+          input: H(Dropdown, {
+            options: markdownOptions,
+            value: mdValue,
+            onchange: (val: string) => setLoadingState({ md: val === "0" }),
```
Comment (styfle) [inline web/index.ts:272]:
```suggestion
            onchange: (val: string) => setLoadingState({ md: val === "1" }),
```

File: web/index.ts [hunk 2]
```diff
@@ -1,419 +1,430 @@
...
+    images = [imageLightOptions[0].value],
+    widths = [],
+    heights = [],
+    showToast = false,
+    messageToast = "",
+    loading = true,
+    selectedImageIndex = 0,
+    overrideUrl = null,
+  } = state;
+  const mdValue = md ? "1" : "0";
```
Comment (styfle) [inline web/index.ts:213]:
```suggestion
  const mdValue = '0';
```

Perhaps this is what you want? To disable markdown always?

Comment (whitep4nth3r) [reply web/index.ts:213]: Ah, thank you!

File: web/index.ts (hunk 1 above)
Comment (whitep4nth3r) [reply web/index.ts:272]: I very much appreciate you taking the time to check in on stream and help me out :)

**Pull request merged**